dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtilityTest.java

```diff
@@ -57,34 +70,75 @@ public void testReadBom_coordinates() throws ArtifactDescriptorException {
     Assert.assertEquals("com.google.cloud:google-cloud-bom:0.61.0-alpha", bom.getCoordinates());
   }
 
+  private static final String UPDATE_GOLDEN_ARTIFACTS = "UPDATE_GOLDEN_ARTIFACTS";
+  private static final Path GOLDEN_FILE =
+      Paths.get(
+          "src/test/resources",
+          getPackageName(RepositoryUtilityTest.class).replace('.', '/'),
+          "goldenBomArtifacts.txt");
+
   @Test
-  public void testReadBom_path() throws MavenRepositoryException, ArtifactDescriptorException {
+  public void testReadBom_path() throws MavenRepositoryException, IOException {
     Path pomFile = Paths.get("..", "boms", "cloud-oss-bom", "pom.xml");
-    
+
     Bom currentBom = RepositoryUtility.readBom(pomFile);
-    Bom oldBom = RepositoryUtility.readBom("com.google.cloud:libraries-bom:2.6.0");
-    ImmutableList<Artifact> currentArtifacts = currentBom.getManagedDependencies();
-    ImmutableList<Artifact> oldArtifacts = oldBom.getManagedDependencies();
-    
+
     String coordinates = currentBom.getCoordinates();
-    Truth.assertThat(coordinates).startsWith("com.google.cloud:libraries-bom:");
-    Truth.assertThat(coordinates).endsWith("-SNAPSHOT");
-    
-    // This is a characterization test to verify that the managed dependencies haven't changed.
-    // However sometimes this list does change. If so, we want to 
-    // output the specific difference so we can manually verify whether
-    // the changes make sense. When they do make sense, we update the test. 
-    if (currentArtifacts.size() != 217) {
-      Truth.assertThat(currentArtifacts).containsExactlyElementsIn(oldArtifacts);
+    assertThat(coordinates).startsWith("com.google.cloud:libraries-bom:");
+    assertThat(coordinates).endsWith("-SNAPSHOT");
+
+    // Assert that the managed dependencies haven't unexpectedly changed.
+    // If this fails, update the golden file so the changes can be reviewed by running
+    // this test with UPDATE_GOLDEN_ARTIFACTS set to any nonempty string. Then
+    // rerun the test without that.
+
+    if (shouldUpdateGoldenArtifactsFile()) {
+      updateGoldenArtifactsFileAndFail(currentBom.getManagedDependencies());
     }
+
+    assertWithMessage(
+            "If BOM artifacts have changed, rerun this test setting the environment variable %s"
+                + " and then try again.",
```

Comment:
```suggestion
                + " and then try again. If you see this in a presubmit, ensure your PR includes the change in goldenBomArtifacts.txt.",
```

How about adding the above? Just in case when I encounter this error in Circle CI for a PR after forgetting to include the file change.

Reply:
Done.